src/wagtailbakery/views.py

```diff
@@ -88,8 +88,9 @@ def build_object(self, obj):
         """
         site = obj.get_site()
         logger.debug("Building %s" % obj)
+        secure_request = site.port == 443 or getattr(settings, 'SECURE_SSL_REDIRECT', False)
```

Comment:
I decided to also check for the site port. I'll add some tests later if you agree with the change.

Reply:
Eh.. It appears to be breaking some tests, but only for some versions of Python.. Anyway, I'll wait for your feedback before looking at it.

Reply:
Thanks for the notice @loicteixeira . In which case are you trying to build pages via ssl? Almost every site runs in http modus, as ssl offloading is done on LB level or page building is done on local machine. In this really necessary?

Reply:
The app is hosted on Heroku and as per their requirement, the [SSL redirection needs to be performed at the application level](https://help.heroku.com/J2R1S4T8/can-heroku-force-an-application-to-use-ssl-tls#django).

Reply:
@robmoorman Any further thoughts on the approach then? Maybe you would prefer it to be a `BAKERY_SECURE_REQUESTS` setting which defaults to `False` instead of the automatic detection?